src/Statistics/StatisticsProcessor.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Statistics;
+
+/**
+ * @internal
+ */
+interface StatisticsProcessor
+{
+    public function processStatistics(): Statistics;
```

Comment:
I guess sometime in the future this would - at least - have `$from` and `$to` arguments?

Reply:
And maybe `createStatistics()` as a more accurate name?

Reply:
yep, it will take some dates in argument... as soon as we need it ;)